packages/config/src/layer2s/arbitrum.ts

```diff
@@ -215,42 +216,36 @@ export const arbitrum: Layer2 = {
   },
   permissions: [
     {
-      name: 'Security Council',
-      accounts: [
-        {
-          address: EthereumAddress(
-            '0x3666a60ff589873ced457a9a8a0aA6F83D708767',
-          ),
-          type: 'MultiSig',
-        },
-      ],
-      description:
+      ...discovery.getGnosisSafeDetails(
+        'SecurityCouncil',
         'The admin of all contracts in the system, capable of issuing upgrades without notice and delay. This allows it to censor transactions, upgrade bridge implementation potentially gaining access to all funds stored in a bridge and change the sequencer or any other system component (unlimited upgrade power). It is also the admin of the special purpose smart contracts used by validators.',
+      ),
+    },
+    {
+      ...discovery.contractAsPermissioned(
+        discovery.getContract('ArbitrumProxyAdmin'),
+        'This contract is an admin of SequencerInbox, RollupEventInbox, Bridge, Outbox, Inbox and ChallengeManager contracts. It is owned by the Upgrade Executor.',
+      ),
     },
```

Comment:
No need to spread if that's the only field
```suggestion
    discovery.contractAsPermissioned(
        discovery.getContract('ArbitrumProxyAdmin'),
        'This contract is an admin of SequencerInbox, RollupEventInbox, Bridge, Outbox, Inbox and ChallengeManager contracts. It is owned by the Upgrade Executor.',
      ),
    
```